lib/common/profile.js

```diff
@@ -30,66 +36,20 @@ function profileServiceFactory(FileLoader, Constants, Promise, _, waterline, Log
      * @extends {FileLoader}
      */
     function ProfileService () {
-        this.loader = new FileLoader();
+        dbRenderable.call(this);
+        this.directory = Constants.Profiles.Directory;
+        this.collectionName = 'profiles';
```

Comment:
Moved the waterline dependency from here to db-renderable, and reluctant to re-include it for the name.  Maybe a constant should be used here and in the models?